Error occurs when building portable class library with MSBuild

# Summary
This issue is related to [this comment](https://github.com/OpenTouryoProject/OpenTouryoTemplates/pull/57#issuecomment-250375745).

When running *3_Build_PortableClassLibrary.bat* in *root_VS2015\programs\C#* folder, the following error occurs:
```txt
C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\AppxPackage\Microsoft.AppXPackage.Targets(283,22): error MSB4086: 条件 "'$(TargetPlatformVersion)'>= '10.0.11000.0'" で、数字ではなく "10.0.11000.0" と評価された "10.0.11000.0" に対して、数値比較を実行しようとしました。 [C:\Users\y-maegawa\Desktop\OpenTouryoTemplates-nishino\root_VS2015\programs\C#\Frameworks\PortableClassLibrary\PCLFramework\PCLFramework.csproj]
```

# Information
When overwriting environment variable *VisualStudioVersion* to *12.0*, this error is eliminated.
(I don't understand the reason why this method enables to eliminate error.)
```bat
set VisualStudioVersion=12.0
```

Please confirm.

--- a/root_VS2015/programs/C#/3_Build_PortableClassLibrary.bat
+++ b/root_VS2015/programs/C#/3_Build_PortableClassLibrary.bat
@@ -15,6 +15,11 @@ set CURRENT_DIR="%~dp0"
 @rem --------------------------------------------------
 call %CURRENT_DIR%z_Common.bat
 
+@rem --------------------------------------------------
+@rem Add environment variable to avoid build error.
+@rem --------------------------------------------------
+set VisualStudioVersion=12.0
+
 rem --------------------------------------------------
 rem Build the batch PortableClassLibrary(PortableClassLibrary)
 rem --------------------------------------------------

